scripts/PMID_downloader.py

```diff
@@ -0,0 +1,90 @@
+import requests
+from Bio import Entrez
+from selenium import webdriver
+from selenium.webdriver.chrome.options import Options
+import time
+
+#This is for using the PubMed API
+Entrez.email = "patrick-simon.nairne@bih-charite.de"
```

Comment:
Never, ever push credentials into git.

You can either load them from a config file or get them from the environment.
Let me know if you have questions